SharpPcap/LibPcap/NativeLibrary.cs

```diff
@@ -5,15 +5,24 @@
 using System.Reflection;
 using System.Runtime.InteropServices;
 
+#if !NET8_0_OR_GREATER
```

Comment:
Request to make this `NET6_0_OR_GREATER`. I know the support ends in November, but not everybody is up to date.
`NativeLibrary` is available since .NET core 3.0.

Reply:
Exactly, support ends in November, everybody that is still using .NET 6 will still be able to use the library, but they won't benefit from trimming support.